src/darkslide/generator.py

```diff
@@ -516,9 +516,11 @@ def render(self):
         html = template.render(context)
 
         if self.embed:
-            images = re.findall(r'url\(["\']?(.*?\.(?:jpe?g|gif|png|svg)[\'"]?)\)', html, re.DOTALL | re.UNICODE)
```

Comment:
Shouldn't you just replace `.*?` with `[^")]*?` or similar? I don't get how your change even fixes the problem :|

Reply:
In a nutshell: the previous regex captures everything until it finds **an image-file extension**, followed by the closing parenthesis (see https://regex101.com/r/pWmN1e/1 for an example of failure).

The fix is quite stupid really:
1. the regex now captures everything until it finds the _first_ closing parenthesis thus matching any file URL (image or not),
2. then see if the file extension does match an image URL and keep only those.

See https://regex101.com/r/nMBsqr/1

To be honest, that fix is far from being perfect, because for instance it would fail if the path includes a parenthesis (see https://regex101.com/r/I75UnR/1). A _real_ fix would be to use an actual X(HT)ML parser (and not just here) - but that's a much bigger work.

Reply:
Thanks, I get it now :) 

Please also update changelog/authors.

Reply:
Err... I'm sorry, I've been reviewing [CONTRIBUTING.rst](https://github.com/ionelmc/python-darkslide/blob/master/CONTRIBUTING.rst) and previous pull-requests, but I can't find a list of contributors to complete?

But it's OK, I'm not looking for credits ;)

I've pushed an update for the changelog.